embassy-boot/boot/src/fmt.rs

```diff
@@ -0,0 +1,225 @@
+#![macro_use]
```

Comment:
Is it possible to use `embassy::fmt` instead of copy-pasting this?

Reply:
I don't think so, I tried to do it once, but I think it wasn't possible, at least at the time. It's present in embassy-nrf, embassy-stm32 and embassy at present, so I think maybe we can fix all of these in one swoop, if possible.